src/react-native/06-building-custom-components/building-custom-components.md

```diff
@@ -52,11 +95,57 @@ If you’ve implemented the solution correctly, the tests will pass when you run
 
 ## Objective 2: Passing props
 
-TODO
+We’ve taken a great step to make our code more readable and our app more maintainable by creating the `StateList` component.
+
+Let’s keep the good refactoring rolling by creating a `ListItem` component to house the JSX used to render each state in the list.
+
+### Using component props
+
+In React, props (short for “properties”) are how we pass data from a parent component to a child component. Since function React components are fundamentally JavaScript functions, you can think of props like the arguments you pass to a function.
+
+Also note that React component props are not the same as the "properties" that exist on a DOM element.
```

Comment:
Any time we mention HTML or the DOM, we should either translate it into something more general or we should preface it with “If you’re familiar with front-end web development…”

For this line, maybe:

> To clarify, props in React Native should not be confused with the properties on HTML elements in web development, even though both share the same name.